lib/Parse/ParseDecl.cpp

```diff
@@ -4587,7 +4592,7 @@ Parser::parseDeclList(SourceLoc LBLoc, SourceLoc &RBLoc, Diag<> ErrorDiag,
   if (RBLoc.isInvalid())
     hadError = true;
 
-  if (!Context.LangOpts.EnableTypeFingerprints)
+  if (L->isCodeCompletion())
```

Comment:
This looks suspicious, because the predicate on line 4559 is different. Should this one be the same?

Reply:
My guess is that previously code completion had EnableTypeFingerprints disabled, so this brings back the old behavior but only when performing code completion. I already suggested Robert file a radar to investigate this further in my review above.